packages/math-input/src/components/icon.tsx

```diff
@@ -55,9 +53,9 @@ class Icon extends React.PureComponent {
                 return (
                     <TextIcon character={icon.data} style={styleWithFocus} />
                 );
+            default:
+                throw new Error("No icon or symbol provided");
         }
-
-        throw new Error("No icon or symbol provided");
```

Comment:
TS automatically removed this, considering it unreachable I suppose. I added it back as part of the switch statement.